### DISCUSSION TITLE
Path/Road VTM

### DISCUSSION THREAD

**Discussion by sfckor** [JohanFalt/Foundry_WoD20#461] [Q&A] [answered]
Is there any way to add Roads/Paths to the sheet? I have an elder who is following a Road not on the list

**Comment by JohanFalt** [comment]
Not at the moment.

https://github.com/JohanFalt/Foundry_WoD20/issues/277

**Comment by JohanFalt** [comment, marked as answer]
Solution for this will come in coming version of v2.5
